### PR TITLE
ddns-scripts: add ddns-sendupdate option

### PR DESCRIPTION
**Pull request by SPiskurskyi**
Added a tool that makes a request to the dynamic DNS service to update information, similar to the curl/wget tools (GET request)

Reference: OWF-2871

### CODE REVIEW

**Comment by septimomend** [comment]
Looks ok, but reassigned for deeper review

**Comment by s-skorobreshchuk** [comment]
Technical implementation is good.
But, changes doesn't make sense for me... We need to remove curl to not provide security breach to perform HTTP requests, and that's why we implemented our own binary that making HTTP requests and that using curl library under the hood....
For me it doesn't fix a security breach, but also adds extra code that we need to maintain potentially (even if it is so easy now)...

@MykolaKopanyshyn @septimomend what do you think? I believe we shouldn't provide any binary that makes HTTP requests, or leave the standard curl if we anyway need it...

File: net/ddns-scripts/src/send-update.c [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+int main(int argc, char *argv[]) {
+    CURL *curl = NULL;
+    CURLcode res;
+	
+    curl = curl_easy_init();
+    if (!curl)
+        return 1;
+
+    curl_easy_setopt(curl, CURLOPT_URL, argv[1]);
```
Comment (MykolaKopanyshyn) [inline net/ddns-scripts/src/send-update.c:12]: Is it working?

Your libcurl program just download provided URL.
Could you provide me an example how send-update is used(params and co).

To my understanding, curl is used to update(i.e. upload) current IP address of the DDNS client to the server.
So, how it is working???

File: net/ddns-scripts/src/send-update.c [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ 
+    res = curl_easy_perform(curl);
+
+    if(res != CURLE_OK)
```
Comment (MykolaKopanyshyn) [inline net/ddns-scripts/src/send-update.c:16]: The main idea of using our binary instead or curl - to remove possibility to download something to the CPE using curl.
So, I would rather expect some kind of hardcoded list of URL we can work with.

**Review by MykolaKopanyshyn — CHANGES_REQUESTED** [review]

File: net/ddns-scripts/src/send-update.c (hunk 1 above)
Comment (SPiskurskyi) [reply net/ddns-scripts/src/send-update.c]: Example of a GET request using a newer version:
ddns-sendupdate "http://gurucam:teiltest@dynupdate.noip.com/nic/update?hostname=lwglo.ddns.net&myip=127.0.0.1"

File: net/ddns-scripts/src/send-update.c (hunk 2 above)
Comment (SPiskurskyi) [reply net/ddns-scripts/src/send-update.c]: done in https://github.com/kn-gl/feed-owf/pull/26

**Comment by SPiskurskyi** [comment]
The main implementation moved to https://github.com/kn-gl/feed-owf/pull/26

**Review by s-skorobreshchuk — APPROVED** [review]

**Pull request merged**